### PR TITLE
feat: add resources for Ethereum

### PR DESCRIPTION
**Pull request by eminmtas**
Just added Ethereum's Whitepaper and some resources about Ethereum

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,18 @@ A curated list of resources to become aligned 🟩
 
 ## Bitcoin
 
-- [Bitcoin: A Peer-to-Peer Electronic Cash System](https://bitcoin.org/bitcoin.pdf) - Bitcoin white paper.
+- [Bitcoin: A Peer-to-Peer Electronic Cash System](https://bitcoin.org/bitcoin.pdf) - Bitcoin Whitepaper
```
Comment (NicolasRampoldi) [inline README.md]: It's written white paper not Whitepaper according to this source: [When to Use a Whitepaper - White Paper Style Guide - LibGuides at University of Massachusetts Lowell](https://libguides.uml.edu/whitepaper_style#:~:text=White%20paper%20is%20more%20widely,white%20paper%20is%20the%20standard.)

File: README.md [hunk 2]
```diff
@@ -3,11 +3,18 @@ A curated list of resources to become aligned 🟩
...
+
+- [A Next-Generation Smart Contract and Decentralized Application Platform](https://ethereum.org/en/whitepaper/) - Ethereum Whitepaper
+- [The history of Ethereum](https://ethereum.org/en/history/)
+- [Ethereum for everyone](https://ethereum.org/en/layer-2/) - Ethereum L2s
+- [Ethereum community hub](https://ethereum.org/en/community/)
 
 ## Eigen Layer
 
-- [EigenLayer: The Restaking Collective](https://docs.eigenlayer.xyz/assets/files/EigenLayer_WhitePaper-88c47923ca0319870c611decd6e562ad.pdf) - EigenLayer white paper.
+- [EigenLayer: The Restaking Collective](https://docs.eigenlayer.xyz/assets/files/EigenLayer_WhitePaper-88c47923ca0319870c611decd6e562ad.pdf) - EigenLayer Whitepaper
```
Comment (NicolasRampoldi) [inline README.md]: See comment above

**Review by NicolasRampoldi — CHANGES_REQUESTED** [review]
Check comments and add full stop to each end of line following the convention already used.

File: README.md (hunk 1 above)
Comment (eminmtas) [reply README.md]: Just corrected it. Thanks for the info, didn't know that

**Comment by eminmtas** [comment]
I think it is ready right now. Thanks for the review.

**Review by NicolasRampoldi — APPROVED** [review]

**Pull request merged**